utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -179,6 +179,9 @@ int main(int argc, char **argv)
     case 31:
         strcpy(funcName, "color_cast");
         break;
+    case 34:
```

Comment:
Lets use the new test suite. We need this to be added in the test_suite folder and verify unit/perf tests.